src/main/java/com/google/maps/GeocodingApiRequest.java

```diff
@@ -86,11 +86,11 @@ public GeocodingApiRequest latlng(LatLng latlng) {
 
   /**
    * Set the bounding box of the viewport within which to bias geocode results more prominently.
-   * This parameter will only influence, not fully restrict, results from the geocoder. (
+   * This parameter will only influence, not fully restrict, results from the geocoder.
    *
    * <p>For more information see <a
-   * href="https://developers.google.com/maps/documentation/geocoding/?hl=pl#Viewports">Viewports
-   * documentation</a>.
+   * href="https://developers.google.com/maps/documentation/geocoding/intro?hl=pl#Viewports">
```

Comment:
While you're here, please would you fix this URL? It should be:
https://developers.google.com/maps/documentation/geocoding/start#Viewports

FYI, the URL parameter "hl=pl" shouldn't be there. It's requesting a version of the page in Polish (which we don't actually have) and is probably the result of someone from Poland making this change while viewing the docs.

To see the language specification in action, try this URL which requests the page in German:
https://developers.google.com/maps/documentation/geocoding/start?hl=de

If you leave the parameter off the URL, the developer docs site will look for language info in the user's browser or as specified in the language option at the bottom of the page.